pkg/cli/prompt/text/text.go

```diff
@@ -84,7 +92,7 @@ func NewTextModel(prompt string, options TextModelOptions) Model {
 		options:   options,
 		prompt:    prompt,
 		textInput: ti,
-	}
+	}, nil
```

Comment:
Can you look into this as an approach? https://github.com/charmbracelet/bubbletea/blob/main/examples/progress-download/tui.go#L44

The framework should automatically send us `WindowSizeMsg`